### PR TITLE
Throw some donuts onto home page

### LINKED ISSUE #30: Create donuts for sectors

### PR DESCRIPTION
**Pull request by jpb7**
`NOTE`: only the first (ie. left-most on screen) `Donut` instance is ready to go as is. The remaining two need to take on some different props.

Each `Donut` component takes three props: `title`, `labels`, and `dataset`.

- The `title` is just what it sounds like. It'll be displayed above the donut.
- `labels` are the distinct fields of the donut. This should be an array.
- The `dataset` is some pre-processed, numeric data that you want to render in the chart.

### CODE REVIEW

File: stock-tracker/src/components/home/Home.js [hunk 1]
```diff
@@ -1,23 +1,76 @@
...
+  );
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const marketCaps = await fetchMarketCaps(bigFive);
+      setMarketCaps(marketCaps);
+    };
+    fetchData()
+  }, [bigFive]);
+
```
Comment (srimel) [inline stock-tracker/src/components/home/Home.js:26]: I wonder if we could move most of this logic into Donut.js? Like if we could just pass in an array of symbols as props and the Donut will take care of fetching marketCaps?

Comment (srimel) [reply stock-tracker/src/components/home/Home.js:26]: Might help with merge conflicts in Home.js as well

Comment (srimel) [reply stock-tracker/src/components/home/Home.js:26]: Talked with @jpb7 and am okay with this design.

**Review by srimel — APPROVED** [review]
Looks good, glad we have another type of graph now!

File: stock-tracker/src/components/home/Home.js (hunk 1 above)
Comment (jpb7) [reply stock-tracker/src/components/home/Home.js:26]: per the discord, idea is to enable each donut instance to take a different set of pre-processed data

**Pull request merged**